doc/js_tutorials/js_assets/js_image_classification.html

```diff
@@ -0,0 +1,255 @@
+        You can change the code in the &lt;textarea&gt; to investigate more.
+    </p>
+    <div>
+        <div class="control"><button id="tryIt">Try it</button></div>
+        <textarea class="code" rows="25" cols="100" id="codeEditor" spellcheck="false"></textarea>
+        <p class="err" id="errorMessage"></p>
+    </div>
+
+    <div>
+        <table cellpadding="0" cellspacing="0" width="0" border="0">
+            <tr>
+                <td>
+                    <canvas id="canvasInput" width="400" height="400"></canvas>
+                </td>
+                <td>
+                    <p id='status' align="left"></p>
+                    <table style="visibility: hidden;" id="result">
+                        <thead>
+                            <tr>
+                                <th scope="col">#</th>
+                                <th scope="col" width=300>Label</th>
+                                <th scope="col">Probability</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <th scope="row">1</th>
+                                <td id="label0" align="center"></td>
+                                <td id="prob0" align="center"></td>
+                            </tr>
+                            <tr>
+                                    <th scope="row">2</th>
+                                    <td id="label1" align="center"></td>
+                                    <td id="prob1" align="center"></td>
+                            </tr>
+                            <tr>
+                                    <th scope="row">3</th>
+                                    <td id="label2" align="center"></td>
+                                    <td id="prob2" align="center"></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                    <p id='time' align="left"></p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <div class="caption">
+                        canvasInput <input type="file" id="fileInput" name="file" accept="image/*">
+                    </div>
+                </td>
+                <td></td>
+            </tr>
+        </table>
+    </div>
+
+    <script src="utils.js" type="text/javascript"></script>
+
+    <script id="codeSnippet" type="text/code-snippet">
+        let labels;
+        if(firstRun){
+            loadLables();
+            firstRun = false;
+        }        
+
+        let inputSize = [224,224];
+        let mean = [104, 117, 123];
+        let std = 1;
+        let input = getBlodFromImage(inputSize, mean, std);
+
+        let modelPath = "bvlc_googlenet.caffemodel";
+        let modelUrl = "https://raw.githubusercontent.com/akineeic/opencv_model/master/image_classification/caffe/bvlc_googlenet.caffemodel";
```

Comment:
This URL links to your own github repo. It might not stable and acceptable for final opencv submission. Could you please leverage opencv model zoo script?